test/integration_tests/run_experiment.py

```diff
@@ -59,6 +59,12 @@ def load_config(scenario_file, model_file, updater_file, dataset_file):
         default=12 * 3600,
         help="Maximum execution time.",
     )
+    parser.add_argument(
```

Comment:
in both files, this argument is considered optional but if not provided, the script will fail

Reply:
Good catch, updated in next version